src/components/input/select/style.css

```diff
@@ -19,14 +21,38 @@
 	align-items: center;
 	cursor: default;
 	padding: .7em .6em;
+	position: relative;
+}
+
+:host::slotted(label) {
```

Comment:
probably want to target sometime like `:host::slotted([slot=label])` instead in case the given label is not a label element. other "smoothly inputs" usually have their own label element so they can properly define the `for` attribute on the tag but in this element that might not be as useful since the current elements it is using is breaking markup rules anyway. But when that is fixed this element probably should contain its own label element.